### DISCUSSION TITLE
Why nativewind v4?

### DISCUSSION THREAD

**Discussion by martmalo** [founded-labs/react-native-reusables#89] [Q&A] [answered]
Currently, I'm using NativeWind v2 in my project, but I came across this library. I was wondering why there is a need for NativeWind v4, as it is unreleased and still in development? Are there any breaking changes from v2 to v4?

**Comment by mrzachnugent** [comment, marked as answer]
The main reason is [CSS variables](https://github.com/mrzachnugent/react-native-reusables/blob/main/apps/starter-base/global.css).

CSS variables allow us to set colours for light and for dark mode and facilitates theming. 

You can find the other advantages here: [NativeWind v4 Announcement](https://www.nativewind.dev/v4/announcement)

**Comment by martmalo** [comment]
Okay, makes sense. Thanks!
